sources/functions/rclone

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Install fuse
+function _fuse() {
```

Comment:
This really doesn’t need to be its own function. It only needs to run on install.